#Sensor Type Constants
string CAMERA=CAMERA
string ANALOG=ANALOG
string DIGITAL=DIGITAL

#Service Message
string platform_name	#Unique Platform Name
string sensor_type		#Type Of Sensor
---
string topic			#Topic To Send Information On
